packages/functions/docs/modules/oidc.md

```diff
@@ -97,3 +98,38 @@ A promise that resolves to the OIDC token.
 #### Defined in
 
 [packages/functions/src/oidc/get-vercel-oidc-token.ts:25](https://github.com/vercel/vercel/blob/main/packages/functions/src/oidc/get-vercel-oidc-token.ts#L25)
+
+---
+
+### getVercelOidcTokenSync
+
+▸ **getVercelOidcTokenSync**(): `string`
+
+Returns the OIDC token from the request context or the environment variable.
+
+This function is used to retrieve the OIDC token from the request context or the environment variable.
+It checks for the `x-vercel-oidc-token` header in the request context and falls back to the `VERCEL_OIDC_TOKEN` environment variable if the header is not present.
+
+context headers.
```

Comment:
Looks like there is a typo here and in the original `getVercelOidcToken` documentation to be cleaned up.